### PR TITLE
feat: add button for dark mode

### PR DESCRIPTION
**Pull request by peasoft**
Fix https://github.com/star-history/star-history/pull/323#issuecomment-1568411210
image

### CODE REVIEW

**Comment by peasoft** [comment]
@boojack @d-bytebase Review please

File: src/components/EmbedMarkdownSection.vue [hunk 1]
```diff
@@ -65,8 +76,28 @@ const embedCode = computed(() => {
 `;
 });
 
+const embedCode2 = computed(() => {
```
Comment (boojack) [inline src/components/EmbedMarkdownSection.vue:79]: How about `embedDarkModeCode`?

File: src/components/EmbedMarkdownSection.vue [hunk 2]
```diff
@@ -65,8 +76,28 @@ const embedCode = computed(() => {
...
+</a>
+`;
+});
+
 const handleCopyBtnClick = () => {
   utils.copyTextToClipboard(embedCode.value);
   toast.succeed("Embed markdown code copied");
 };
+
+const handleCopyBtnClick2 = () => {
```
Comment (boojack) [inline src/components/EmbedMarkdownSection.vue:99]: Let's use `handleDarkModeCopyBtnClick`.

**Review by boojack — APPROVED** [review]
LGTM, some nits. And thanks for your contribution!

**Pull request merged**